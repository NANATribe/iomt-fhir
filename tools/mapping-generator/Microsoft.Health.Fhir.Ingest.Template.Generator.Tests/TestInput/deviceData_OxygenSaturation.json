{
  "type": "Oxygen Saturation",
  "time": "2022-07-01T12:02:00Z",
  "device": "TestDeviceId",
  "patient": "TestPatientId",
  "encounter": "TestEncounter",
  "correlation": "TestCorrelation",
  "values": {
    "oxygenPercentage": 99
  }
}