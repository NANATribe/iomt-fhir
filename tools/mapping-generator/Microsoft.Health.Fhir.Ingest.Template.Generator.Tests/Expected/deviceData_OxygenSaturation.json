{
  "templateType": "CalculatedContent",
  "template": {
    "typeMatchExpression": "$..[?(@.type=='Oxygen Saturation')]",
    "deviceIdExpression": "$.device",
    "patientIdExpression": "$.patient",
    "encounterIdExpression": "$.encounter",
    "timestampExpression": "$.time",
    "correlationIdExpression": "$.correlation",
    "values": [
      {
        "valueName": "oxygenPercentage",
        "valueExpression": "$.values.oxygenPercentage"
      }
    ],
    "typeName": "Oxygen Saturation"
  }
}